tests/test_dataframe_histograms.py

```diff
@@ -0,0 +1,28 @@
+import unittest
+import ROOT, PyRDF
```

Comment:
I'm not seeing a reason why this commit is coupled with the new constructors as they are different things. Yes, these tests are possible because you added the new constructors, but I would say that it's still 2 different things (but I will allow it if @etejedor agrees :) )

Also, from what we discussed on last week's meeting, you should provide a test that is an exact replica of what is currently in ROOT. This means, no `tdf = PyRDF.RDataFrame(64)`.
What we discussed was that you should find a way of replacing  ROOT.ROOT.RDataFrame with pyRDF dynamically. From my understanding, this would be to re-use (when this project gets merged into ROOT), the tests from RDataFrame. Some ideas that were discussed (I don't know how/if they work, sorry):
 * create a ROOT module, with the RDF replaced by PyRDF, and prepend it to python path (then when importing ROOT this would be the one being picked, but we were not sure this worked)
 * find a way to replace this globally in the tests configurations.

Can you have a look to see if you find any way of achieving this?
Thanks

Reply:
@diocas . Yes. Fair enough that the 2 tests could've been a part of a new PR :) 

And if I remember correctly, we discussed the replacing of `ROOT.ROOT.RDataFrame` for the files under `$ROOTSYS/tutorials/dataframe`. Because, at least one of the tests in this PR require the use of PyROOT's `ROOT.ROOT.RDataFrame`. Like here : https://github.com/shravan97/PyTDF/pull/19/files#diff-7ef345175f2df21f8bef48e12ae5ac46R8

Am I right about this @diocas and other mentors ?

Reply:
I think what was discussed in the end was to just replace the import by something like `from PyRDF import RDataFrame as ROOT.ROOT.RDataFrame`. No big deal though, since in the end PyRDF will become ROOT.ROOT.RDataFrame. What is important is that tests pass.